plugins/CoreHome/angularjs/report-export/reportexport.popover.html

```diff
@@ -62,7 +62,7 @@
 
     <div class="col l12" ng-show="showUrl">
         <textarea piwik-select-on-focus readonly class="exportFullUrl">{{ getExportLinkWithoutToken() }}</textarea>
-        <div class="tooltip" ng-bind-html="'CoreHome_ExportTooltipWithLink'|translate:'<a target=_blank href=\'/index.php?module=UsersManager&action=userSecurity\'>':'</a>':'ENTER_YOUR_TOKEN_AUTH_HERE'"></div>
+        <div class="tooltip" ng-bind-html="'CoreHome_ExportTooltipWithLink'|translate:'<a target=_blank href=\'./index.php?module=UsersManager&action=userSecurity\'>':'</a>':'ENTER_YOUR_TOKEN_AUTH_HERE'"></div>
```

Comment:
you could also have `href='index.php?...` or just `href='?module=...` (the `.` is unnecessary)

Reply:
thanks, done